envergo/templates/moulinette/loi_sur_leau/result_soumis.html

```diff
@@ -3,8 +3,8 @@
 <h3>Marche à suivre</h3>
 
 <ul>
-  <li>Un dossier de déclaration Loi sur l’eau (également nommé « déclaration IOTA ») doit être constitué et <a href="https://entreprendre.service-public.fr/vosdroits/R62929" target="_blank" rel="noopener">déposé en ligne</a>.
-</li>
+  <li>Un <a href="https://entreprendre.service-public.fr/vosdroits/R62929" target="_blank" rel="noopener">dossier de déclaration Loi sur l’eau (également nommé « déclaration IOTA ») doit être constitué et déposé en ligne</a>.
```

Comment:
sur ce contenu-là je pense que ça fait trop, c'est moins lisible quand c'est souligné, il ya une parenthèse et des guillemets (je dois préciser IOTA car le site service public ne fait aucune mentin de Loi sur l'eau)
En outre, "constituer le dossier" ne se fait pas en ligne, c'est le BE qui le fait. Le lien est juste pour déposer le dossier en ligne.
J'en resterais à ma proposition "être déposé en ligne"